### PR TITLE
Connect front and back end + camera 

### PR DESCRIPTION
**Pull request by VladLiteanu**
Connected the front and back end together.
Most of the logic is taken care of in js. 
Some changes have been made to "main.js" to allow importing. 
"smartcar.ino" updated to receive inputs.
Related issues: #22

### CODE REVIEW

**Comment by VladLiteanu** [comment]
Added camera support to the car. For now, it just transmits binary to the broker.

File: front_end/Scripts.js [hunk 1]
```diff
@@ -1,32 +1,101 @@
...
+        let gp = navigator.getGamepads()[0];
+        if (gp !== null){
+            gp.buttons.forEach(button => {
+                if(button.pressed == true){
+                    console.log(button, gp.buttons.indexOf(button));
+                    client.publish('/smartcar/control/buttons', String(gp.buttons.indexOf(button)))
+                }
+            })
+            /*
+            if(gp.axes[0] > 0.15 || gp.axes[0] < -0.15) {
```
Comment (TaofikArnouk) [inline front_end/Scripts.js:70]: That is pretty good, do we need left joystick to control anything for the car or do we take out of the code permanently? Or should we keep it if we are going to add something in the future.

Comment (VladLiteanu) [reply front_end/Scripts.js:70]: We only use the right joystick to control the car for now so that part is not needed for now. 
Should it be deleted?

Comment (TaofikArnouk) [reply front_end/Scripts.js:70]: I guess we can delete it and if needed we can add it again later

**Review by TaofikArnouk — APPROVED** [review]
I have nothing else to add. We can add the left joystick if needed in the future.

**Pull request merged**